app/src/main/java/org/hackillinois/android/view/OnboardingActivity.kt

```diff
@@ -0,0 +1,48 @@
+package org.hackillinois.android.view
```

Comment:
This wasn't working for me when I was testing locally -- can we go over this during the meeting?